<launch>
	<node pkg="topic_tools" type="mux" name="mux_cmd_vel"
		args="/cmd_vel /nav/cmd_vel /ocu/cmd_vel /nifti_teleop_joy/cmd_vel /local_joy/cmd_vel" >
		<remap from="/mux/select" to="/private/mux_cmd_vel/select" />
		<remap from="/mux/add" to="/private/mux_cmd_vel/add" />
		<remap from="/mux/delete" to="/private/mux_cmd_vel/delete" />
		<remap from="/mux/list" to="/private/mux_cmd_vel/list" />
		<remap from="/cmd_vel_select" to="/private/mux_cmd_vel/deprecated" />
		<remap from="/mux/selected" to="/mux_cmd_vel/selected" />
	</node>

	<node pkg="nifti_teleop" type="nifti_mux_ctrl.py" name="nifti_mux_control" output="screen"/>
</launch>
